qemu/tests/cfg/blockdev_snapshot_install.cfg

```diff
@@ -51,3 +51,11 @@
        enable_ceph_unattended = no
        enable_ceph_sn1 = no
        enable_ceph_image1 = yes
+    nbd:
+       enable_nbd_cd1 = no
+       enable_nbd_unattended = no
+       enable_nbd_sn1 = no
+       enable_nbd_image1 = yes
+       nbd_port_image1 = 10811
```

Comment:
Do we want to add image_format_image1 = raw here to limit the target installation image format?

Reply:
No need to add it, we set image_format=raw in our kar cmdline for system disk.